ci-operator/step-registry/ipi/ovirt/pre/e2e/ipi-ovirt-pre-e2e-chain.yaml

```
@@ -1,7 +1,8 @@
 chain:
-  as: ipi-ovirt-pre
+  as: ipi-ovirt-pre-e2e
```

```suggestion
  as: ipi-ovirt-pre-monitoringpvc
```
or `-withcsi` or no postfix at all. This chain doesn't run e2e in any case

---

Done, I removed the prefix